### DISCUSSION TITLE
Chinese leecher infinitely downloading a torrent

### DISCUSSION THREAD

**Discussion by loonsies** [anacrolix/torrent#904] [General]
Hello,
We are a small community that shares a abandonware japanese game from the 90s. We have a torrent list with different content from that game. One of the torrents of the list have been downloaded hundreds of times by the same client recently. The peer uses anacrolix/torrent.

![image](https://github.com/anacrolix/torrent/assets/37041587/a4b05d12-cc2a-4cf4-ac39-69ca85033247)

after blocking them on qbittorrent, they eventually came back a few hours after 

![image](https://github.com/anacrolix/torrent/assets/37041587/37fa719b-cc51-49af-bb8f-d13d38f7c599)

(the torrent involved is only about 300MB)

They stay at 0% progress and keep downloading for days.
Is this a bug of your client or is this like an attack? I'm confused...

**Comment by anacrolix** [comment]
See https://github.com/anacrolix/torrent/issues/889. There was a bug that could manifest this, however I think this might be a malicious peer. See also the linked discussion linked from the issue where people are talking about it.
